### PR TITLE
Fix FileNotFound when predicting gas stations

### PR DESCRIPTION
**Pull request by Baschdl**
As assumed the error happens when we try to predict the price for a gas station where we don't have any data for it. @WGierke Do you have an idea how to best integrate the possibility that a prediction can return None, 0, nothing etc. without breaking the whole prediction and FixedPathGasStation logic?

### CODE REVIEW

**Comment by WGierke** [comment]
I'm not quite sure about that. What's the plan if we don't have training data? Interpolate the prediction using the k-NN gas stations?

**Comment by Baschdl** [comment]
Just exclude it completely?

**Comment by Baschdl** [comment]
I don‘t think we have to incorporate gas stations with missing data into the price prediction along a route. In the case of predicting the price for a single gas station, the data would be provided.

**Comment by Baschdl** [comment]
@feeds Is this ready for reviewing? Then please remove the `WIP` label.

File: src/serving/route_prediction.py [hunk 1]
```diff
@@ -57,7 +57,11 @@ def _get_gas_station_points_near_path(path, radius=0.02):
 def predict_price(id, time):
     model, _, df_forecast = train_and_predict(gas_station_id=id, start_time=time, end_time=time, use_cached=True,
                                               cache=True)
-    deci_cent = round(Decimal(df_forecast.loc[0, 'yhat']), 0)
+    if df_forecast is None:
```
Comment (WGierke) [inline src/serving/route_prediction.py]: Is this really a clean solution? Wouldn't e.g. -1 express better that the prediction we give is wrong?

**Pull request merged**